hudi-client/hudi-client-common/src/main/java/org/apache/hudi/keygen/CommonTimestampBasedKeyGenerator.java

```diff
@@ -46,15 +42,11 @@
 
 import static java.util.concurrent.TimeUnit.MILLISECONDS;
 import static java.util.concurrent.TimeUnit.SECONDS;
-import static org.apache.hudi.keygen.KeyGenUtils.DEFAULT_PARTITION_PATH;
-import static org.apache.hudi.keygen.KeyGenUtils.EMPTY_RECORDKEY_PLACEHOLDER;
-import static org.apache.hudi.keygen.KeyGenUtils.NULL_RECORDKEY_PLACEHOLDER;
 
 /**
- * Key generator, that relies on timestamps for partitioning field. Still picks record key by name.
+ * Common Key generator, that relies on timestamps for partitioning field. Still picks record key by name.
  */
-public class TimestampBasedKeyGenerator extends SimpleKeyGenerator {
-
+public class CommonTimestampBasedKeyGenerator extends CommonSimpleKeyGenerator {
```

Comment:
all the `Common` generators. are avro based. So let's name them like `SimpleAvroKeyGenerator` , `TimestampBasedAvroKeyGenerator` ?

Reply:
> all the `Common` generators. are avro based. So let's name them like `SimpleAvroKeyGenerator` , `TimestampBasedAvroKeyGenerator` ?

done